packages/mobile-sdk/lib/devDependencies.json

```
@@ -34,9 +34,24 @@
     "version": "0.76.6",
     "devOnly": true
   },
-  "@react-native-community/cli": {
-    "name": "@react-native-community/cli",
-    "version": "^15.1.2",
+  "@rnef/cli": {
+    "name": "@rnef/cli",
+    "version": "^0.5.0",
```

lets update packages before merging this PR :D 
```suggestion
    "version": "^0.5.2",
```